### ISSUE TITLE
DNS resolver configuration in reverse proxy should respect /etc/resolv.conf for k8s etc.. container runtime.

### ISSUE THREAD

**Issue by knaoe** [dfoxg/kratos-admin-ui#122]
**Describe the bug**
When I deploy this image on my Kubernetes cluster, the reverse proxy fails due to a DNS lookup error.
I didn't notice this problem because I didn't test it on kubernetes when I tested 2.0.0 in #119 🙇 .

**Expected behavior**
The system should resolve the hostname for Kratos in the internal network, applicable not only to Docker locally but also to other container runtimes.
Nginx should prioritize using /etc/resolv.conf, and default to 127.0.0.11 only if this configuration file is unavailable.
I confirmed /etc/resolv.conf is also available in the container on Docker for Desktop.
```
# cat /etc/resolv.conf 
nameserver 127.0.0.11
options ndots:0
```

**Kubernetes:**
 - Version 1.21

**kratos-admin-ui**
 - Version 2.0.0

**Related context**
[It seems nginx-ingress-controller is also taking same approach.](https://github.com/search?q=repo%3Akubernetes%2Fingress-nginx%20resolv.conf&type=code)

**Comment by knaoe** [comment]
I will make a PR if I find some spare time in the coming days.

**Comment by knaoe** [comment]
@dfoxg I made a PR #124 ! Could you please test it?

**Comment by dfoxg** [comment]
Thanks for that contribution! I just released v2.0.1 🎉
